test/change_health/request/report_test.rb

```diff
@@ -101,5 +101,27 @@ class ReportTest < Minitest::Test
         end
       end
     end
+
+    describe '#delete_report' do
+      describe 'get response confirming deletion' do
+        let(:report_name) { 'X3000000.XX' }
+        let(:response) { build_response(file: "claim/report/report.#{report_name}.delete.response.json") }
+        let(:single_report_endpoint) { ChangeHealth::Request::Claim::Report::ENDPOINT + "/#{report_name}" }
+
+        before do
+          stub_change_health(endpoint: single_report_endpoint, response: response, verb: :delete)
+
+          @response = claim_report.delete_report(report_name, headers: report_headers)
```

Comment:
I feel i should be using instance variables more often in tests, this is pretty convenient! 👏